mw-webapp/src/logic/wayPage/reportsTable/modalContentJobTags/ModalContentJobTags.module.scss

```diff
@@ -1,6 +1,6 @@
 @import "src/styles/variables";
 
-$ModalJobTagMaxWidth: 150px;
+$ModalJobTagMaxWidth: 298px;
```

Comment:
if modal width 298px then on mobile with 320px how it will looks? And strange value - why 298 why not 299 or 297 and etc

Reply:
Refactored. But are we making a beautiful website or a beautiful code? :)

With a block width of 300px, only one column is created, not two.
Therefore, I decided to make the most of the space, but at the same time make two columns.

![bug_04_01__300](https://github.com/user-attachments/assets/0786a344-192a-46bc-8250-cd8348b99341)

![bug_04_01__290](https://github.com/user-attachments/assets/d86ba5cf-ad0d-430f-8ae0-75d31da794bd)